io.github.CatalystDevOrg.Catalyst/io.github.CatalystDevOrg.Catalyst.json

```
@@ -0,0 +1,64 @@
+{
+  "app-id": "io.github.CatalystDevOrg.Catalyst",
+  "base": "org.electronjs.Electron2.BaseApp",
+  "base-version": "23.08",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "23.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "run.sh",
+  "separate-locales": false,
+  "finish-args": [
+    "--share=ipc",
+    "--share=network",
+    "--socket=wayland",
+    "--socket=x11",
+    "--socket=pulseaudio",
+    "--device=dri",
+    "--talk-name=org.freedesktop.Notifications",
+    "--talk-name=org.kde.StatusNotifierWatcher",
+    "--talk-name=org.gtk.Notifications",
+    "--own-name=io.github.CatalystDevOrg.Catalyst"
```

```suggestion
    "--talk-name=org.gtk.Notifications"
```

this is granted by default.